code/modules/transport/pods/ships.dm

```diff
@@ -814,15 +814,18 @@ ABSTRACT_TYPE(/obj/structure/vehicleframe)
 	bound_height = 64
 	view_offset_x = 16
 	view_offset_y = 16
+	speedmod = 0.9
 	//luminosity = 5 // will help with space exploration
 	var/maxboom = 0
 
 	onMaterialChanged()
+		if (src.material)
+			src.speedmod /= (1 / (1 - (src.material.getProperty("electrical") - 5) / 15))
```

Comment:
Why are we doing this before and after the parent call? Is this supposed to reset it or something? Why not initial?

Reply:
It's to reset it yeah. Not using initial() because there's other sources of where the speedmod can be modified